### PR TITLE
Create nuget for Microsoft.CodeAnalysis.Remote.Workspaces...

### PR DESCRIPTION
**Pull request by KevinH-MS**
Creating a nuget package that I can reference.  Eventually, I imagine this will be included in some sort of "Microsoft.CodeAnalysis.Next" nupkg.

### CODE REVIEW

**Comment by KevinH-MS** [comment]
FYI @tannergooding @heejaechang @srivatsn 
/CC @dotnet/roslyn-analysis

File: src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <description>
+      A shared package used by the .NET Compiler Platform ("Roslyn") including support for coordinating analysis of projects and solutions using a separate server process. Do not install this package manually, it will be added as a prerequisite by other packages that require it.
+
+Supported Platforms:
+- .NET Framework 4.5
+    </description>
+    <dependencies>
+      <dependency id="Microsoft.CodeAnalysis.Common" version="[$version$]" />
+      <dependency id="Microsoft.CodeAnalysis.Workspaces.Common" version="[$version$]" />
+      <dependency id="Microsoft.Composition" version="$MicrosoftCompositionVersion$" />
```
Comment (heejaechang) [inline src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec]: I think RemoteWorkspace also has dependency to Workspace.Desktop.

Comment (KevinH-MS) [inline src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec]: It surprised me, but I believe that is part of Microsoft.CodeAnalysis.Workspaces.Common currently (there is no separate Desktop nuget).

**Comment by heejaechang** [comment]
I am not expert on nuget. so probably @tannergooding is right person to sign off. I looks fine to me though.

File: src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.CodeAnalysis.Remote.Workspaces</id>
```
Comment (tannergooding) [inline src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec]: Wouldn't our normal naming convention be `Microsoft.CodeAnalysis.Workspaces.Remote`?

File: src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <licenseUrl>$licenseUrl$</licenseUrl>
+    <projectUrl>$projectUrl$</projectUrl>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <tags>$tags$</tags>
+    <serviceable>true</serviceable>
+  </metadata>
+  <files>
+    <file src="Microsoft.CodeAnalysis.Remote.Workspaces.dll" target="lib\portable-net45+win8" />
+    <file src="Microsoft.CodeAnalysis.Remote.Workspaces.xml" target="lib\portable-net45+win8" />
+    <file src="Microsoft.CodeAnalysis.Remote.Workspaces.dll" target="lib\net45" />
```
Comment (tannergooding) [inline src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec]: Are we actually `net45` compatible? I thought we had some `netstandard1.3` and/or `net46` dependencies?

**Comment by KevinH-MS** [comment]
Also FYI @shyamnamboodiripad for acquisition...

File: src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec (hunk 3 above)
Comment (KevinH-MS) [inline src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec]: I've copied the targets from Microsoft.CodeAnalysis.Workspaces.Common (they should be the same).

File: src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec (hunk 2 above)
Comment (KevinH-MS) [inline src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec]: I've matched the dll name.

**Comment by tannergooding** [comment]
LGTM

**Comment by KevinH-MS** [comment]
@shyamnamboodiripad pointed out there are no actual public types in this assembly, so there's no point in publishing to nuget.  I've updated to just publish to myget.

File: src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec (hunk 1 above)
Comment (shyamnamboodiripad) [inline src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec]: There is a bug to split up MS.CA.Workspaces.Common #12444

**Comment by shyamnamboodiripad** [comment]
👍

**Pull request merged**